challenge.rb

```diff
@@ -1,26 +1,33 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  input.map { |x| x.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  dog = input.find_all { |x| x.downcase == "dog"}
+  dog
```

Comment:
the `dog` variable isn't doing anything here; you can cut it, just like in the problem above